CONTRIBUTING.md

```diff
@@ -8,22 +8,20 @@ Development should take place in the Docker container.
 
 ### Setting up the docker container
 
-You can build the docker image locally using:
+Navigate your local `refinebio-examples` repository.
+Now, you can build the docker image locally using:
 ```
-docker build -< Dockerfile -t refinebio-examples
+docker build -< Dockerfile -t ccdl/refinebio-examples
 ```
 Or by pulling the image from Docker hub.
 ```
 docker pull ccdl/refinebio-examples
 ```
-
-Navigate to your local `refinebio-examples` repository.
-
-Then run this command:
+Then run this command to start up a container:
 ```
-docker run --mount type=bind,target=/home/rstudio,source=<LOCAL_PATH_TO_REFINEBIO-EXAMPLES_REPO> -e PASSWORD=<PASSWORD> -p 8787:8787 refinebio-examples
+docker run --mount type=bind,target=/home/rstudio,source=$PWD -e PASSWORD=eevee -p 8787:8787 ccdl/refinebio-examples
```

Comment:
Do you want to note people can change the password?

Reply:
Shoot! I accidentally copied and pasted that in there.

Reply:
now everybody knows your secrets 🧐